views/app.html

```diff
@@ -44,85 +44,77 @@
   </div>
 </div>
 
-<div class="content create-something">
-  <div class="row">
-    <div class="col-xs-10 col-xs-offset-1 col-lg-6 col-lg-offset-0 col-lg-push-6 section-content">
-      <div class="col-xs-12 section-header">{{ gettext("Become a Maker") }}</div>
-      <div class="col-xs-12 col-lg-8 section-text add-margin">{{ gettext("Strips away complexity") }}</div>
-      <div class="col-lg-12 desktop-install-app">
-        <div class="col-lg-3 desktop-install-app-btn-wrapper pull-left">
-          <button class="btn btn-block desktop-install-app-btn">{{ gettext("Get App")}}</button>
-        </div>
-        <div class="col-lg-12 send-install-link-container hidden pull-left">
-          <div class="col-lg-12 phone-number-label pull-left">
-            <div class="col-lg-4 pull-left">
-              <i class="fa icon-mobile-phone"></i> {{ gettext("Enter your phone number")}}
-            </div>
-          </div>
-          <div class="col-lg-4 pull-left">
-            <input class="phone-num-input" type="tel" name="" value="">
-          </div>
-          <div class="col-lg-4">
-            <button class="btn btn-block send-sms-btn">{{ gettext("Send Text")}}</button>
-          </div>
-          <div class="col-lg-12 message-sent-container hidden pull-left">
-            <span>{{ gettext("message sent") }}</span>
-          </div>
-          <div class="col-lg-12 message-sent-error-container hidden pull-left">
-            <span>{{ gettext("message sent error") }}</span>
-          </div>
-        </div>
-        <div class="col-lg-12 pull-left">
-          <div class="col-lg-8 pull-left">
-            <div class="phone-error-popup off">{{ gettext("phone format error") }}</div>
-          </div>
-        </div>
-        <div class="col-lg-12 sms-privacy-message pull-left hidden">
-          <div class="col-lg-8 pull-left">
-            {{ gettext("sms privacy message") | safe }}
+{% macro preview(image) %}
```

Comment:
Make sure you include the `jpg` versions somehow. They're supposed to be in place of the gifs before they load.